### PR TITLE
chore: remove unused methods from server side

### PR DESCRIPTION
**Pull request by dgozman**
Most of these were api methods that are no longer called directly.

### CODE REVIEW

File: src/server/electron.ts [hunk 1]
```diff
@@ -125,17 +115,16 @@ export class ElectronApplication extends EventEmitter {
...
-    await this.evaluate(({ app }) => app.quit());
+    await this._nodeElectronHandle!.evaluate(({ app }) => app.quit());
     this._nodeConnection.close();
     await closed;
   }
 
-  async waitForEvent(event: string, optionsOrPredicate: types.WaitForEventOptions = {}): Promise<any> {
-    const options = typeof optionsOrPredicate === 'function' ? { predicate: optionsOrPredicate } : optionsOrPredicate;
-    const progressController = new ProgressController(this._timeoutSettings.timeout(options));
+  async waitForEvent(event: string, predicate?: Function): Promise<any> {
```
Comment (JoelEinbinder) [inline src/server/electron.ts:123]: I'm surprised to see waitForEvent still here. It got put on the clientside elsewhere.

Comment (JoelEinbinder) [reply src/server/electron.ts:123]: Maybe it should be _waitForEvent?

File: src/rpc/client/consoleMessage.ts [hunk 2]
```diff
@@ -19,8 +19,7 @@ import { JSHandle } from './jsHandle';
 import { ConsoleMessageChannel, ConsoleMessageInitializer } from '../channels';
 import { ChannelOwner } from './channelOwner';
 
-let __dummyInitializer: ConsoleMessageInitializer;
-type ConsoleMessageLocation = typeof __dummyInitializer.location;
+type ConsoleMessageLocation = ConsoleMessageInitializer['location'];
```
Comment (JoelEinbinder) [inline src/rpc/client/consoleMessage.ts:22]: Pro TypeScript Refactor :+1:

**Review by JoelEinbinder — APPROVED** [review]

File: src/rpc/client/consoleMessage.ts (hunk 2 above)
Comment (dgozman) [reply src/rpc/client/consoleMessage.ts:22]: 😄

File: src/server/electron.ts (hunk 1 above)
Comment (dgozman) [reply src/server/electron.ts:123]: Yeah, I can make it private in a follow up!

**Pull request merged**